docker-make

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-docker_tag=73c9e07
+docker_tag=latest
```

Comment:
prefer a named tag. `latest` lets the docker image wreck your dev environment behind your back.  the docker image should only be updated intentionally and along with any required compatibility changes